apache/superset
# feat(explore): Redesign of Run/Save buttons

### SUMMARY
This PR moves the "Save" button to chart header and "Run" button (renamed to "Create/Update chart") to the bottom of control panel.

Full list of changes:
- "Save" button moved to the right side if chart header
- Renamed "Run" button to "Create chart" if it's a new chart (no query has been made yet) or "Update chart" if queries were run before on that chart
- Moved "Create/Update chart" button to the bottom of control panel - it's sticky, meaning it always stays in place when user scrolls the control panel
- "Data/Customize" tab bar is sticky - only the content of tab gets scrolled, the tab bar stays in place
- Error icon moved from next to "Run/Save" buttons to "Data" tab header
- Change empty state message when creating chart and all controls have been added - "Your chart is ready to go!"

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/15073128/161961889-643b4ec0-97fb-4a9d-9b6c-56a0174f6498.mov


### TESTING INSTRUCTIONS
1. Create new chart
2. Verify that the button at the bottom of control panel says "Create chart" and is disabled
3. Add necessary controls
4. "Create chart" button should not be disabled now + the message in empty state changed to a prompt to click the button
5. Click "Create chart" button
6. Verify that query was run and the name of the button changed to "Update chart"
7. Verify that "Save" button in the chart header works as before

### ADDITIONAL INFORMATION



@kasiazjc
